jobmaster/cvsJob/cvsJobMaster.go

```diff
@@ -95,13 +95,20 @@ func (jm *JobMaster) InitImpl(ctx context.Context) error {
 		}
 		jm.syncFilesInfo[workerID] = &workerInfo{file: file, curLoc: 0, handle: nil}
 	}
-	jm.status = lib.WorkerStatusNormal
-	return jm.UpdateJobStatus(ctx, jm.Status())
+	return nil
 }
 
 func (jm *JobMaster) Tick(ctx context.Context) error {
 	filesNum := 0
 	jm.counter = 0
+	if jm.status != lib.WorkerStatusNormal {
```

Comment:
if  ！= workerstatusnormal  ,should return error at once

Reply:
if != normal, it's supposed to be init. I set it in "Init" function at first, but meet `update status` retry fail multiple times. So I retry `update status` in Tick function